kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -157,8 +157,9 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
 
   def getBatchSessionList(batchType: String, from: Int, size: Int): Seq[Session] = {
     allSessions().filter {
-      case batchSession: KyuubiBatchSessionImpl =>
+      case batchSession: KyuubiBatchSessionImpl if batchType != null =>
```

Comment:
by the way, I am not sure if no specified batchType, its default values is `null` or empty string?

```
Option(batchType).exists(_.nonEmpty)
```

Reply:
it is null